### PR TITLE
[Feature] Add french documentation

### PR DESCRIPTION
**Pull request by survivant**
# Description

I create documentation in french and fix some links in the documentation

Maybe it will have to be updated after this PR : https://github.com/usebruno/bruno/pull/757

### Contribution Checklist:

- [X] **The pull request only addresses one issue or adds one feature.**
- [X] **The pull request does not introduce any breaking changes**
- [ ] **I have added screenshots or gifs to help explain the change if applicable.**
- [X] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [ ] **Create an issue and link to the pull request.**

### CODE REVIEW

File: contributing_fr.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+[English](contributing.md) | [Українська](/contributing_ua.md) | [Русский](/contributing_ru.md) | [Türkçe](/contributing_tr.md) | [Deutsch](/contributing_de.md) | **Français**
+
+## Améliorons Bruno ensemble !!
+
+Je suis ravi que vous souhaitiez améliorer Bruno. Voici les directives pour commencer à faire fonctionner Bruno sur votre ordinateur.
+
+### Stack Technologique
```
Comment (j0k3r) [inline contributing_fr.md]:
```suggestion
### Stack Technique
```

File: docs/development_fr.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+[English](/docs/development.md) | [Українська](/docs/development_ua.md) | [Русский](/docs/development_ru.md) | [Türkçe](/contributing_tr.md) | [Deutsch](/docs/development_de.md) | **Français**
+
+## Développement
+
+Bruno est en cours de développement en tant qu'application de bureau. Vous devez lancer l'application en exécutant l'application Next.js dans un terminal, puis exécuter l'application Electron dans un autre terminal.
```
Comment (j0k3r) [inline docs/development_fr.md]:
```suggestion
Bruno est développé en tant qu'application de bureau. Vous devez lancer l'application en exécutant l'application Next.js dans un terminal, puis exécuter l'application Electron dans un autre terminal.
```

File: docs/development_fr.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+### Développement Local
+
+```bash
+# Utilisez la version 18 de NodeJS
+nvm use
+
+# Installer les dépendances
+npm i --legacy-peer-deps
+
+# Construire le module grapgql-docs
```
Comment (j0k3r) [inline docs/development_fr.md]:
```suggestion
# Construire le module graphql-docs
```

File: readme_fr.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+[![GitHub version](https://badge.fury.io/gh/usebruno%2Fbruno.svg)](https://badge.fury.io/gh/usebruno%bruno)
+[![CI](https://github.com/usebruno/bruno/actions/workflows/unit-tests.yml/badge.svg?branch=main)](https://github.com/usebruno/bruno/workflows/unit-tests.yml)
+[![Commit Activity](https://img.shields.io/github/commit-activity/m/usebruno/bruno)](https://github.com/usebruno/bruno/pulse)
+[![X](https://img.shields.io/twitter/follow/use_bruno?style=social&logo=x)](https://twitter.com/use_bruno)
+[![Website](https://img.shields.io/badge/Website-Visit-blue)](https://www.usebruno.com)
+[![Download](https://img.shields.io/badge/Download-Latest-brightgreen)](https://www.usebruno.com/downloads)
+
+[English](/readme.md) | [Українська](/readme_ua.md) | [Русский](/readme_ru.md) | [Türkçe](/readme_tr.md) | [Deutsch](/readme_de.md) | **Français**
+
+Bruno est un nouvel et innovant client API, conçu pour révolutionner le statu quo représenté par Postman et des outils similaires.
```
Comment (j0k3r) [inline readme_fr.md]:
```suggestion
Bruno est un nouveau et innovant client API, conçu pour révolutionner le statu quo représenté par Postman et outils similaires.
```

File: readme_fr.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+Bruno stocke vos collections directement dans un dossier sur votre système de fichiers. Nous utilisons un langage de balisage en texte brut, le Bru, pour enregistrer des informations sur les requêtes API.
+
+Vous pouvez utiliser git ou tout système de contrôle de version de votre choix pour collaborer sur vos collections API.
+
+Bruno est hors ligne uniquement. Il n'y a aucun plan pour ajouter une synchronisation dans le cloud à Bruno, jamais. Nous accordons de l'importance à la confidentialité de vos données et pensons qu'elles devraient rester sur votre appareil. Lisez notre vision à long terme [ici](https://github.com/usebruno/bruno/discussions/269)
```
Comment (j0k3r) [inline readme_fr.md]:
```suggestion
Bruno fonctionnement uniquement en hors ligne. Il n'y a aucun plan pour ajouter une synchronisation dans le cloud. Nous accordons de l'importance à la confidentialité de vos données et pensons qu'elles devraient rester sur votre appareil. Lisez notre vision à long terme [ici](https://github.com/usebruno/bruno/discussions/269)
```

File: readme_fr.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+- [Hub de Connaissances](https://github.com/usebruno/bruno/discussions/386)
+- [Scriptmania](https://github.com/usebruno/bruno/discussions/385)
+
+### Soutien ❤️
+
+Wouf ! Si vous aimez le projet, appuyez sur le bouton ⭐ !!
+
+### Partagez vos témoignages 📣
+
+Si Bruno vous a aidé au travail et à vos équipes, n'oubliez pas de partager vos [témoignages sur notre discussion GitHub](https://github.com/usebruno/bruno/discussions/343)
```
Comment (j0k3r) [inline readme_fr.md]:
```suggestion
Si Bruno vous a aidé dans votre travail et/ou vos équipes, n'oubliez pas de partager vos [témoignages](https://github.com/usebruno/bruno/discussions/343)
```

File: readme_fr.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+### Publication dans de nouveaux gestionnaires de paquets
+
+Veuillez consulter [ici](publishing.md) pour plus d'informations.
```
Comment (j0k3r) [inline readme_fr.md]:
```suggestion
Veuillez consulter [ceci](publishing.md) pour plus d'informations.
```

**Comment by helloanoop** [comment]
Hey @survivant There was another PR that I just merged for the french documentation. Can you review that make any changes as needed ?

**Comment by survivant** [comment]
Yes, I'll

Le mer. 25 oct. 2023, 13 h 12, Anoop M D ***@***.***> a
écrit :

**Comment by survivant** [comment]
@helloanoop done.

**Pull request closed without merging**